src/validations/content.ts

```diff
@@ -1,6 +1,6 @@
 import { Avatar, EntityType, Profile } from '@dcl/schemas'
 import { fromErrors, Validation } from '../types'
-import { validationAfterADR45, validationGroup } from './validations'
+import { validationAfterADR100, validationAfterADR45, validationGroup } from './validations'
```

Comment:
I suggest you create an issue in ADR, this way you reserve the number (ADR number => issue number) and then no one else will collide with your chosen id.